about.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/about.css">
+    <title>About GSMCT 2022</title>
+    <!-- Fonts -->
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header class="about-header">
+
+        <div class="header-mobile">
+            <div class="overlay">
+            </div>
+            <div class="hamburger-icon">
+                <div class="top-bar"></div>
+                <div class="middle-bar"></div>
+                <div class="bottom-bar"></div>
+            </div>
+            <nav class="mobile-menu hide">
+                <ul>
+                    <li class="links-container"><a href="index.html">Home</a></li>
+                    <li class="links-container"><a href="about.html" class="active">About</a></li>
+                </ul>
+            </nav>
+
+        </div>
+        <div class="header-desktop">
+            <nav class="desktop-menu">
+                <div class="logo-container">
+                    <a href="index.html"><img src="images/logo.png" alt="Logo of the event organizers"></a>
+                </div>
+                <ul>
+                    <li class="links-container"><a href="index.html">Home</a></li>
+                    <li class="links-container"><a href="about.html" class="active">About</a></li>
+                </ul>
+            </nav>
+        </div>
```

Comment:
- [ ] The navbar in the desktop view is divided in a top dark bar and a bottom white bar. As you can see from the screenshot below there is a top dark bar with some links. Kindly make changes accordingly. :pray: 

![image](https://user-images.githubusercontent.com/64894927/132554687-e4bf6bc1-235c-418b-b1b4-6a704740004c.png)

Reply:
Sorry, I had to be absent from Thursday to Saturday last week. So I'm repeating the week after having used only a single attempt. 
That change is done now.